pkg/patch/cmd.go

```diff
@@ -26,6 +26,7 @@ type patchArgs struct {
 	format        string
 	output        string
 	bkOpts        buildkit.Opts
+	updateAll     bool
```

Comment:
i am okay with `--update-all` but checking whether we want to call something different like `--upgrade-all` (since it technically upgrades packages)

Reply:
removed flag, just checking for report flag now